README.md

```diff
@@ -1 +1,142 @@
+
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+ 
+  <br/>
+
 # Portfolio-Microverse
+
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse Project <a name="about-project"></a>
+
+**Hello Microverse Project** is a simple Hello World repository, configured with linters for html and css
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML</a></li>
+     <li><a href="#">CSS</a></li>
+  </ul>
+</details>
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Linter for HTML**
+- **LInter for CSS**
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+<ul>
+    <li>Web Browser</li>
+     <li>Node.js installed</li>
+  </ul>
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/donice/Portfolio-Microverse.git
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+
+
+```sh
+npx hint .
+  npx stylelint "**/*.{css/scss}"
+```
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/donice)
+- Twitter: [@twitterhandle](https://twitter.com/drdonice)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/doniceubaru)
+- LinkedIn: [V .](https://www.linkedin.com/in/doniceubaru/)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Responsive Sites**
+- [ ] **Dynamic Sites**
+- [ ] **Database driven dynamic Sites**
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse team.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT) licensed.
```

Comment:
- [ ] As suggested by the previous reviewer you did awesome work by giving the correct file name for the license. But please always make sure that your link is correct by cross-checking, as I can see that you forget to add the proper name with an extension which again gives a 404-page error. Kindly follow the suggestion to fix it quickly.

```suggestion
This project is [MIT](./MIT.md) licensed.
```